src/components/app.jsx

```diff
@@ -1,18 +1,35 @@
 import React from 'react';
 import Header from './header/header';
-import contacts from '../data';
+import contactsData from '../data';
 import ToolBar from './toolBar/toolBar';
 import ContactList from './contactList/contactList';
 
 
 export default class App extends React.PureComponent {
+  state = {
+    contacts: contactsData,
+  };
+
+  handleInputChange = (evt) => {
+    const value = evt.target.value;
+
+    const filteredContacts = contactsData.filter(contact => {
```

Comment:
You can just `contact => contact.name.includes(value)`

Reply:
done